src/main/webapp/i18n/en/student-dashboard.json

```
@@ -33,7 +33,8 @@
             "manage": "Manage",
             "studentView": "Student view",
             "general": {
-                "noDataFound": "No data found."
+                "noDataFound": "No data found.",
+                "noExercisesFoundWithAppliedFilter": "No exercises were found to which the selected filters apply."
```

No strong opinion here - just an idea:

```suggestion
                "noExercisesFoundWithAppliedFilter": "No exercises match the selected filters."
```